core/src/main/scala/kafka/metrics/clientmetrics/ClientMetricsConfig.scala

```diff
@@ -0,0 +1,137 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package kafka.metrics.clientmetrics
```

Comment:
@hachikuji @mjsax @AndrewJSchofield Do you think we should either move the classes to parent `metrics` package itself or something else. I didn't change the package created by @sarat-kakarla.

Reply:
I would move to the parent `metrics` package.

Reply:
I have moved it to parent `metrics` package.